### PR TITLE
Update some spaces and fix start command

### PR DESCRIPTION
**Pull request by hyknn**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,109 +4,106 @@ DOCKER_PHP_CONTAINER_EXEC := $(DOCKER_COMPOSE) exec php
...
 	$(DOCKER_PHP_CONTAINER_EXEC) composer install --working-dir=src --no-progress --no-suggest --prefer-dist
 	$(DOCKER_PHP_CONTAINER_EXEC) ./src/vendor/bin/phpunit --dont-report-useless-tests ./tests/bb-modules/
 
-build:          ## Build App with Docker CI image
-    docker run --rm \
-        --mount type=bind,source=$(PWD),target=/opt -w /opt \
-        -e TRAVIS_TAG \
-        fordnox/docker-builder-ci \
-        make release
+build: ## Build App with Docker CI image
```
Comment (jaapmarcus) [inline Makefile:60]: Are we going to use Travis for CI or not.

If not we should remove it from make file

If yes we need to update the main image last time it was 2 years ago

File: Makefile [hunk 2]
```diff
@@ -4,109 +4,106 @@ DOCKER_PHP_CONTAINER_EXEC := $(DOCKER_COMPOSE) exec php
...
+	docker run --rm \
+	--mount type=bind,source=$(PWD),target=/opt -w /opt \
+	-e TRAVIS_TAG \
+	fordnox/docker-builder-ci \
+	make release
 
-release:        ## App release
```
Comment (jaapmarcus) [inline Makefile:71]: Same here

File: Makefile [hunk 3]
```diff
@@ -4,109 +4,106 @@ DOCKER_PHP_CONTAINER_EXEC := $(DOCKER_COMPOSE) exec php
...
+	ant release
 
 revbump:
-    set -e ;\
-        git tag --sort=v:refname | tail -1
-        NEW_VERSION=$$( git tag --sort=v:refname | tail -1 | awk 'BEGIN{FS=OFS="."}{print $$1,$$2,$$3+1}' ) ;\
-        echo "unstable release $$NEW_VERSION" ;\
-        git tag $$NEW_VERSION
-        git push --tags
+	set -e ;\
```
Comment (jaapmarcus) [inline Makefile:75]: If we keep release with this method we probally need to update or remove it

File: Makefile [hunk 4]
```diff
@@ -4,109 +4,106 @@ DOCKER_PHP_CONTAINER_EXEC := $(DOCKER_COMPOSE) exec php
...
+	git push --tags
 
 minorbump:
-    set -e ;\
-        git tag --sort=v:refname | tail -1
-        NEW_VERSION=$$( git tag --sort=v:refname | tail -1 | awk 'BEGIN{FS=OFS="."}{print $$1,$$2+1,0}' ) ;\
-        echo "unstable release $$NEW_VERSION" ;\
-        git tag $$NEW_VERSION
-        git push --tags
+	set -e ;\
```
Comment (jaapmarcus) [inline Makefile:83]: If we keep release with this method we probally need to update or remove it

File: Makefile [hunk 5]
```diff
@@ -4,109 +4,106 @@ DOCKER_PHP_CONTAINER_EXEC := $(DOCKER_COMPOSE) exec php
...
-    # run `make build` before running this target
-    # mysql user: admin pass: admin
-    # access phpmyadmin http://localhost/phpmyadmin/
-    docker container stop boxbilling &>/dev/null || true
-    docker container rm -f boxbilling &>/dev/null || true
-    docker run -i -t --name boxbilling -p "80:80" \
-        -v ${PWD}/build/source:/app \
-        -e MYSQL_ADMIN_PASS=admin \
-        mattrayner/lamp:latest
+	set -e ;\
```
Comment (jaapmarcus) [inline Makefile:91]: If we keep release with this method we probally need to update or remove it

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
Some remarks about the docker file it self. 

Changes look fine but if possible we need to d a cleanup as well

File: Makefile (hunk 1 above)
Comment (evrifaessa) [reply Makefile:60]: We ran into issues with Travis CI in BoxBilling and removed it. The leftovers can also be removed. I think GitHub Actions are enough for us now, if it's not, we can always set up CircleCI or something else in the future.

**Comment by jaapmarcus** [comment]
If should be save  to merge and then we will update it.

**Pull request merged**